### ISSUE TITLE
ConfirmDialog: Default to appendTo="@(body)"

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#5569]
1) Environment
- PrimeFaces version: 8.0-SNAPSHOT
- Does it work on the newest released PrimeFaces version? Version? Yes

## 2) Expected behavior

Using a non global confirm dialog should default to appendTo="@(body)".

## 3) Actual behavior

Default has no appendTo default.  We have a scenario like this in PrimeFaces 7 it is rendering the modal on top but in PrimeFaces 8.0 testing it is rendering the dialog behind the modal.  Manually putting appendTo="@(body)" fixes it but I think for confirm dialog that should be the default.

![image](https://user-images.githubusercontent.com/4399574/73392137-3c7cb980-42a7-11ea-9641-09b73f02f111.png)

![image](https://user-images.githubusercontent.com/4399574/73392163-48687b80-42a7-11ea-9ecc-7dd27ad7f5f2.png)

https://github.com/primefaces/primefaces/blob/4380e49adb34c4afd3b321000607652541db9ba6/src/main/resources/META-INF/resources/primefaces/dialog/dialog.js#L636-L638

## 5) Sample XHTML

```xml
<h:form id="theForm">
           <p:commandButton value="Show" type="button" onclick="PF('confirmDialog').show();" />

            <p:confirmDialog id="dialog" widgetVar="confirmDialog" message="Destroy The World?" modal="true" severity="alert" header="Test" closeOnEscape="true">
                <p:commandButton value="Yes" type="button" styleClass="ui-confirmdialog-yes" icon="pi pi-check" />
                <p:commandButton value="No" type="button" styleClass="ui-confirmdialog-no" icon="pi pi-times" />
            </p:confirmDialog>      
</h:form>
```

I think we should make switch it to an OR.
```javascript
if (!cfg.appendTo || cfg.global) {
    cfg.appendTo = '@(body)';
}
```

**Comment by tandraschko** [comment]
Why did you close the issue itself?
I just talked about the fix itself that if 7.0 worked fine without @(body), it should do the same in 8.0

**Comment by melloware** [comment]
Yep but I can't figure out why my exact scenario is breaking.  I wrote a little PrimeFaces test project doing what I thought was the exact same thing and I can't reproduce it by itself.  I can flip the Test back and forth between 7.0 and 8.0 and it works fine.

 So I am thinking you are right its something in my CSS or project causing it so I figured we should close for now until I can reproduce it standalone. I just can't figure out what it is.
